roxie/ccd/ccdquery.cpp

```diff
@@ -593,6 +593,7 @@ class CQueryFactory : implements IQueryFactory, implements IResourceContext, pub
             else
                 return createRoxieServerDiskReadActivityFactory(id, subgraphId, *this, helperFactory, kind, node, remoteId);
         }
+        case TAKspillread:
```

Comment:
Could possible common up this case with the previous case section.

Reply:
this is the correct action to take, so I don't see any benefit.